apps/user-portal/src/app.tsx

```diff
@@ -67,7 +68,7 @@ export const App = (): ReactElement => {
         dispatch(setServiceResourceEndpoints<ServiceResourceEndpointsInterface>(Config.getServiceResourceEndpoints()));
         dispatch(setI18nConfigs<I18nModuleOptionsInterface>(Config.getI18nConfig()));
         dispatch(setUIConfigs<UIConfigInterface>(Config.getUIConfig()));
-    }, []);
+    }, [dispatch]);
```

Comment:
Shall we remove `dispatch` from the deps array. Even though we get an exhaustive dependency lint warning, adding functions in the array would be problematic and might need some extensive testing.

Check other places as well.